crates/sui-indexer-alt-graphql/schema.graphql

```diff
@@ -631,6 +631,36 @@ type CommandEdge {
 	node: Command!
 }
 
+"""
+A value produced or modified during command execution.
+
+This can represent either a return value from a command or an argument that was mutated by reference.
+"""
+type CommandOutput {
+	"""
+	The transaction argument that this value corresponds to (if any).
+	"""
+	argument: TransactionArgument
+	"""
+	BCS-encoded representation of the value.
+	"""
+	value: Base64
```

Comment:
`value` should be a `MoveValue` -- does gRPC indicate the type of the value (🤞 I'm hoping it does because it uses the `Bcs` type which includes a type name).